shared/checks/oval/no_user_host_based_files.xml

```diff
@@ -0,0 +1,22 @@
+<def-group oval_version="5.11">
+  <definition class="compliance" id="no_user_host_based_files" version="1">
+    <metadata>
+      <title>No .shosts file deployed on the system</title>
+      <affected family="unix">
+        <platform>multi_platform_rhel</platform>
+      </affected>
+      <description>There should not be any .shosts files on the system.</description>
+    </metadata>
+    <criteria operator="AND">
+      <criterion test_ref="test_no_shosts" negate="true" />
+    </criteria>
+  </definition>
+  <unix:file_test check="all" check_existence="at_least_one_exists" comment="look for .shosts in /" id="test_no_shosts" version="1">
```

Comment:
change `check_existence="at_least_one_exists"` to `check_existence="none_exist"`